dandiapi/api/management/commands/create_dev_dandiset.py

```diff
@@ -6,28 +6,52 @@
 from django.contrib.auth.models import User
 from django.core.files.uploadedfile import SimpleUploadedFile
 import djclick as click
+import faker
```

Comment:
This dependency isn't installed anywhere besides in `tox.ini` as part of testing, so this script would not run even if the user has run `pip install .`.

Reply:
Included it in [15ca09b](https://github.com/dandi/dandi-archive/pull/2272/commits/15ca09bb8f4287cf7d97d0247fa96ad74a2e62f2) and force-pushed. If we decide not to keep the augmentation of that management command i'll just drop the commit.